dev-tools/reconfigurator-cli/tests/output/cmds-mupdate-update-flow-stdout

```diff
@@ -64,14 +85,18 @@ set sled 98e6b7c2-2efa-41ca-b20a-0a4d61102fe6 mupdate override: unset -> 6123eac
 set sled 2b8f0cb3-0295-4b3c-bc58-4fe88b57112c mupdate override: unset -> error
 
 
+> # Also set its SP update, which will not be cleared.
```

Comment:
Should this be cleared if there's a mupdate override error? IIRC a mupdate override error means sled-agent won't launch any zones or write new host OS images; should Nexus keep trying to update its SP?

Reply:
Sure. I think in practice this doesn't really matter: a mupdate and a Nexus-driven MGS update are going to interfere with each other without support involvement in some way anyway, and after performing a mupdate all of the prechecks that a pending MGS update performs will fail, so Nexus won't try to do anything.

However, since we wouldn't add a new pending MGS update until we recovered from the sled mupdate, it seems a little cleaner to clear any existing ones?